main/src/main.ts

```diff
@@ -260,8 +260,15 @@ app.on("open-file", (event, filePath) => {
     if (activeWindow()) {
         activeWindow().webContents.send("open-file", filePath)
     } else if (process.platform.includes("darwin")) {
-        const argsToUse = [...process.argv, filePath]
-        createWindow(argsToUse, process.cwd())
+        const argsToUse = [...process.argv.slice(1), filePath]
+
+        if (app.isReady()) {
+            createWindow(argsToUse, process.cwd())
+        } else {
+            app.on("ready", async () => {
```

Comment:
@psxpaul thanks so much for the fix 👍 , One quick note is that the `app.on("ready")` callback is an async function in your change but we dont await `createWindow` which I believe is sync, its I think, a minor point but is it intentional?

Reply:
Thanks for the feedback! I removed the async keyword and re-tested.